src/docfx/lib/JsonUtility.cs

```diff
@@ -135,6 +143,34 @@ public static (List<Error>, JToken) ValidateNullValue(this JToken token, JTokenS
             return (errors, token);
         }
 
+        public static IEnumerable<Error> ValidateSchemaAgainstType(this JToken token, Type type)
```

Comment:
>ValidateSchemaAgainstType [](start = 41, length = 25)

Add a TODO to cache type?

We can add another method to accept JSON schema string later. e.g. for config, the user input and strong type model is different, as there's a conversion in between. Type doesn't work in this case.